src/btt.c

```diff
@@ -337,6 +337,33 @@ read_info(struct btt *bttp, struct btt_info *infop)
 }
 
 /*
+ * btt_is_zero -- (internal) returns true if map_entry is in zero state
+ */
+static inline int
+btt_is_zero(uint32_t map_entry)
+{
+	return (map_entry & ~BTT_MAP_ENTRY_LBA_MASK) == BTT_MAP_ENTRY_ZERO;
+}
+
+/*
+ * btt_is_error -- (internal) returns true if map_entry is in error state
+ */
+static inline int
+btt_is_error(uint32_t map_entry)
+{
+	return (map_entry & ~BTT_MAP_ENTRY_LBA_MASK) == BTT_MAP_ENTRY_ERROR;
+}
+
+/*
+ * btt_is_initial -- (internal) returns true if map_entry is in initial state
+ */
+static inline int
+btt_is_initial(uint32_t map_entry)
```

Comment:
A nit: these functions operate on map entries so should have `map` in their name.  Since they are internal, having `btt` is their name really isn't necessary.  So how about:

*  `map_entry_is_zero()`
*  `map_entry_is_error()`
* `map_entry_is_initial()`